lib/cisco_node_utils/interface_ospf.rb

```diff
@@ -214,6 +216,46 @@ def dead_interval=(interval)
                  @interface.name, '', interval.to_i)
     end
 
+    def default_bfd
+      config_get_default('interface_ospf', 'bfd')
+    end
+
+    # CLI can be either of the following or none
+    # ip ospf bfd
+    # ip ospf bfd disable
+    def bfd
+      val = config_get('interface_ospf', 'bfd', @interface.name)
+      return default_bfd if val == default_bfd
+      val.include?('disable') ? false : true
+    end
+
+    # interface %s
+    #   %s ip ospf bfd %s
+    def bfd=(val)
+      Feature.bfd_enable
```

Comment:
`destroy` will call this setter regardless. This also needs a check for `'no' && not Feature.bfd_enabled`